src/test/mir-opt/array_index_is_temporary.main.SimplifyCfg-elaborate-drops.after.mir.64bit

```diff
@@ -37,9 +37,6 @@ fn main() -> () {
         StorageLive(_6);                 // scope 4 at $DIR/array-index-is-temporary.rs:16:25: 16:26
         _6 = _3;                         // scope 4 at $DIR/array-index-is-temporary.rs:16:25: 16:26
         _5 = const foo(move _6) -> bb1;  // scope 4 at $DIR/array-index-is-temporary.rs:16:21: 16:27
-                                         // ty::Const
-                                         // + ty: unsafe fn(*mut usize) -> u32 {foo}
```

Comment:
The exact type of functions can sometimes be useful.

Reply:
The type is still shown right below (though we should probably cut out the `val` part of the `literal` field as it's useless for zsts.